src/modules/sdl2/consumer_sdl2.c

```diff
@@ -804,8 +808,10 @@ static void *consumer_thread(void *arg)
             // Play audio
             init_audio = consumer_play_audio(self, frame, init_audio, &duration);
 
-            // Determine the start time now
-            if (self->playing && init_video) {
+            // Start the video thread unconditionally. We cannot check for self->playing,
+            // because the variable may be initialized with a delay by a backdround SDL
+            // thread and cause race condition
+            if (init_video) {
```

Comment:
Since `playing` is now only set and never read, it should be removed along with the comment here that no longer makes sense. 

I think this change is OK even though I did not reproduce it in melt with a 240 fps video. Maybe more repro steps are required. Otherwise, I tested it successfully in melt and Shotcut.

Reply:
Hi, @ddennedy!

To reproduce the issue one needs to start/stop hi-fps playback constantly and very quickly. Like:

1) Start 100fps playback
2) Wait 1000ms
3) Stop for 100ms
4) Start again
5) Repeat

This situation is quite easy to reproduce in Krita's GUI, when the user restarts playback by clicking mouse very quickly.

PS:
I'll implement the technical part of the comment now.

Reply:
The flag is now removed!